test/napi/lib/hello.js

```diff
@@ -6,4 +6,11 @@ describe('hello', function() {
     assert.equal(addon.greeting, "Hello, World!");
     assert.equal(addon.greeting, addon.greetingCopy);
   });
+
+  it('should export global singletons for JS primitives', function () {
+    assert.equal(addon.undefined, undefined);
```

Comment:
Can you add an additional assert that `addon.hasOwnProperty('undefined')`? Otherwise, this would still pass with code removed from `lib.rs`.